resource-managers/kubernetes/lightbend-build/README.md

```diff
@@ -0,0 +1,38 @@
+# Instructions
+
+This project adds small programs to test Spark specific features.
+To build the jar run: `sbt assembly`
+This will create the following assembly:
+checkpointing-test-assembly-1.0-SNAPSHOT.jar
```

Comment:
Should we use the same version string we use in releases or perhaps "latest" between releases? This is something we should discuss in the standup.

Reply:
Yes whatever makes sense. Just random version there.

Reply:
Will use latest.